packages/components/forma-36-react-timepicker/package.json

```diff
@@ -16,19 +16,14 @@
     "lint": "tsdx lint"
   },
   "peerDependencies": {
-    "react": ">=16"
+    "react": ">=16.8",
+    "react-dom": ">=16.8"
```

Comment:
Do we only support `16.8` or anything above `16`?

Reply:
I thought because we use hooks we want 16.8

Reply:
ahhh wait these components don't use hooks, right?

Reply:
Good point about the hooks, I thought it was much earlier than `16.8` 
we can keep it for consistency or fallback to `16` if we want to be accurate, up to you.